setup.py

```diff
@@ -15,6 +15,7 @@
     zip_safe=False,
     install_requires=[
         'BeautifulSoup4',
+        'django>=1.10',
```

Comment:
I think this should work, however I'm not sure if a manual check is required, like djangorestframework does with python version [djangorestframework setup.py](https://github.com/encode/django-rest-framework/blob/master/setup.py)